components/prism-neon.js

```diff
@@ -0,0 +1,39 @@
+Prism.languages.neon = {
+	'comment': {
+		pattern: /#.*/,
+		greedy: true
+	},
+	'datetime': {
+		pattern: /(^|[[{(=,\s])\d\d\d\d-\d\d?-\d\d?(?:(?:[Tt]| +)\d\d?:\d\d:\d\d(?:\.\d*)? *(?:Z|[-+]\d\d?(?::?\d\d)?)?)?(?=$|[\]}),\s])/,
```

Comment:
Upon researching this, I found a very similar pattern [here](https://github.com/matej21/neon-js/blob/master/src/decoder.js#L298). Could be please explain why the last `:` is optional in your version? This is the only real difference between the two patterns.

Reply:
It is bug in that library, because regexp should match `'2016-06-03 19:00:00 +0200'`. I'll send PR.